app/models/organization_user.rb

```diff
@@ -1,4 +1,8 @@
 class OrganizationUser < ActiveRecord::Base
   belongs_to :organization
   belongs_to :user
+
+  def invite_mail(invited_by)
```

Comment:
```invite_mail``` feels a little awkward to me. Would ```mail_invite``` (using a verb) make more sense here to make it clear we are sending an email?